apps/frontend/src/features/ai-ask/components/ChatHistorySection.tsx

```
@@ -61,26 +62,36 @@ export const ChatHistorySection = ({ items, expandedIds, onToggle }: ChatHistory
-              <span css={questionTextStyle(theme)}>{item.question}</span>
-              <span css={statusStyle(theme)} aria-label={`답변 상태: ${statusLabel}`}>
-                {statusLabel}
-              </span>
-            </button>
+                <button
+                  type="button"
+                  css={toggleButtonStyle(theme)}
+                  onClick={() => onToggle(item.id)}
+                  onKeyDown={e => handleKeyDown(e, item.id)}
+                  aria-expanded={isExpanded}
+                  aria-controls={answerId}
```

Biome (2.3.14)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)